testui/elements/testui_element.py

```diff
@@ -205,41 +208,59 @@ def visible_for(self, seconds=1):
                            f'for {seconds}s')
         return self
 
-    def wait_until_visible(self, seconds=10.0, log=True):
+    def wait_until_visible(self, seconds=10.0, log=True) -> 'Elements':
         start = time.time()
+        
         err = None
+
         err_text = 'not'
         if self.__is_not:
             err_text = 'is'
+        
         i = 0
```

Comment:
maybe consider renaming `i` variable to something like `entered_loop` and making it a bool

Reply:
I don't see the need for it, so I think it can be deleted

Reply:
i think its there to ensure that even if user sets `seconds=0` or `seconds=-1` or some other absurd value, the the `while` loop is entered at least once

Reply:
Yea, that could be the case. Will add it back then